bin/lib/fetchContent.js

```diff
@@ -0,0 +1,91 @@
+const fetch = require('node-fetch');
+const debug = require('debug')('bin:lib:fetchContent');
+
+const     extractUuid = require('./extract-uuid');
+const    parseRSSFeed = require('./parse-rss-feed');
+const individualUUIDs = require('./individualUUIDs');
+
+const CAPI_KEY = process.env.CAPI_KEY;
+if (! CAPI_KEY ) {
+	throw new Error('ERROR: CAPI_KEY not specified in env');
+}
+
+const CAPI_PATH = 'http://api.ft.com/enrichedcontent/';
+
+function parseRssItemToItem( rssItem ){
+	const itemData = {
+		content : rssItem.description[0],
+		title   : rssItem.title[0],
+		guid    : rssItem.guid[0],
+		pubdate : rssItem.pubDate[0], // <-- NB this should be the now time,
```

Comment:
`<-- NB this should be the now time` Should it? What if it's an older, evergreen article that's been picked for audio-fying at a later date?